tooltester/src/main/java/io/dockstore/tooltester/runWorkflow/WorkflowRunner.java

```diff
@@ -488,6 +488,7 @@ public void uploadAndSaveRunInfo() {
         if (getTotalWallClockTimeInISO861Standard() != null) {
             runMetrics.setExecutionTime(getTotalWallClockTimeInISO861Standard());
         }
+        runMetrics.setDateExecuted(workflowStartTime.toString());
```

Comment:
It's possible for `workflowStartTime` to be `null` so should check it before using it. I believe `getTotalWallClockTimeInISO861Standard()` in the if statement above checks if the `workflowStartTime` is `null` somewhere in the function (double check), so you may just be able to move this into the if statement above

Reply:
Confirmed and added to the if statement above. https://github.com/dockstore/dockstore-support/blob/c642dc5c1f3aabf9dc502d5dc7d00401524834cc/tooltester/src/main/java/io/dockstore/tooltester/runWorkflow/WorkflowRunner.java#L348